### PR TITLE
Move Strand_Init() earlier in wsmanclient code.

### PR DESCRIPTION
**Pull request by palladia**

### CODE REVIEW

**Comment by palladia** [comment]
@paulcallen

File: Unix/wsman/wsmanclient.c [hunk 1]
```diff
@@ -1189,6 +1189,12 @@ MI_Result WsmanClient_New_Connector(
     MI_Result miresult;
     MI_Boolean secureTransport = MI_FALSE;
     MI_Boolean privacy = MI_FALSE;
+    Probable_Cause_Data cause;
+
+    // Most errors here are allocation errors
+    cause.type = ERROR_WSMAN_QUOTA_LIMIT;
+    cause.description = MI_T("Batch allocation failed");
```
Comment (paulcallen) [inline Unix/wsman/wsmanclient.c]: "Out of memory" would be more understandable.

File: Unix/wsman/wsmanclient.c [hunk 2]
```diff
@@ -1323,15 +1348,21 @@ MI_Result WsmanClient_New_Connector(
             /* We don't yet implement utf-16 BOM. Fail */
 
             miresult = MI_RESULT_INVALID_PARAMETER;
-            goto finished;/* We cannot add a UTF-16 BOM to the front yet so need to fail */
+            cause.type = ERROR_WSMAN_OPTIONS_INVALID_VALUE;
+            cause.description = MI_T("Packet encoding of UTF-16 is not yet implemented");
```
Comment (paulcallen) [inline Unix/wsman/wsmanclient.c]: "... not supported" rather than "... not yet implemented"

File: Unix/wsman/wsmanclient.c (hunk 1 above)
Comment (palladia) [inline Unix/wsman/wsmanclient.c]: Fixed.

File: Unix/wsman/wsmanclient.c (hunk 2 above)
Comment (palladia) [inline Unix/wsman/wsmanclient.c]: Fixed.

**Comment by palladia** [comment]
@paulcallen Fixed per comments.

**Pull request merged**